package/duo-pinmux/duo-pinmux.mk

```diff
@@ -0,0 +1,14 @@
+DUO_PINMUX_SITE = $(TOPDIR)/package/duo-pinmux/src
```

Comment:
Usually people will not directly put the source code into the buildroot, they store it in a git server or simply github instead. Are you interested in creating a new repo and modify this `mk` to download the source code from the github? I will then try putting our repos into the organization milkv-duo.

Reply:
I'm new to buildroot, so thank you for guidance in this matter. Sure, I'll create repo for source code, and modify package files to use it.